blueocean-github-pipeline/pom.xml

```diff
@@ -55,6 +55,10 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>fluent-hc</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
```

Comment:
These plugins should not both independently bundle this library. Talk to @oleg-nenashev about it.

Reply:
@jglick  httpclient is a dependency is fluent-hc. For some reason it's not visible resulting in explicitly adding this dependency. Why this would be a problem? Specially since parent pom makes sure both are same version. If it's a trouble I could exclude it from fluent-hc. 

@oleg-nenashev you know what could be causing this? Or how to get around? I could of course exclude it from fluent-hc.

Reply:
I think we are good. Don't see double inclusion of httpclient. This is what's printed with `mvn dependency:tree`:

```
[INFO] +- org.apache.httpcomponents:fluent-hc:jar:4.5.3:compile
[INFO] |  \- commons-logging:commons-logging:jar:1.2:provided
[INFO] +- org.apache.httpcomponents:httpclient:jar:4.5.3:compile
```

Reply:
Use [Apache HttpComponents Client 4.x API Plugin for Jenkins](https://github.com/jenkinsci/apache-httpcomponents-client-4-api-plugin), Luke

Reply:
Done